docs/guide/rolldown.md

```diff
@@ -145,11 +145,19 @@ Thanks to Rolldown and Oxc, various internal Vite plugins, such as the alias or
 
 To test them, you can set the `experimental.enableNativePlugin` option to `true` in your Vite config.
 
-### `@vitejs/plugin-react-oxc`
+### Utilizing Oxc's React refresh transform
 
-When using `@vitejs/plugin-react` or `@vitejs/plugin-react-swc`, you can switch to the `@vitejs/plugin-react-oxc` plugin, which uses Oxc for React's fast-refresh instead of Babel or SWC. It is designed to be a drop-in replacement, providing better build performance and aligning with the underlying architecture of `rolldown-vite`.
+`@vitejs/plugin-react` v5.0.0+ uses Oxc's React refresh transform. If you are not using any Babel plugins (including the React compiler), the full transform would now be done by Oxc and will improve the build performance without any changes other than updating `@vitejs/plugin-react`.
 
-Be aware that you can only switch to `@vitejs/plugin-react-oxc` if you are not using any Babel or SWC plugins (including the React compiler), or mutate the SWC options.
+If you are using `@vitejs/plugin-react-swc` without SWC plugins and custom SWC options, you can switch to the `@vitejs/plugin-react` plugin to utilize Oxc.
+
+::: details `@vitejs/plugin-react-oxc` plugin is deprecated
+
+Previously, we recommended using `@vitejs/plugin-react-oxc` to utilize Oxc's React refresh transform.
+However, we have merged the implementation into `@vitejs/plugin-react` so that it is easier to switch to `rolldown-vite`.
+`@vitejs/plugin-react-oxc` is now deprecated and will no longer be updated.
```

Comment:
```suggestion
Previously, we recommended using `@vitejs/plugin-react-oxc` to utilize Oxc's React refresh transform. However, we have merged the implementation into `@vitejs/plugin-react` so that it is easier to switch to `rolldown-vite`. `@vitejs/plugin-react-oxc` is now deprecated and will no longer be updated.
```